sysutils/pbsecret/Portfile

```
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    roosto pbsecret 9e91917de001a73fc01aa8ba96c5c83c4b7f5a34
+version         20200219
+revision        0
+
+categories      sysutils
+maintainers     @midnightrocket \
```

The details for each maintainer are normally enclosed in curly braces, like so:

```suggestion
maintainers     {@midnightrocket} \
                openmaintainer
```